### PR TITLE
Add a basic JSON validation CI action

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

**Review by ssilverman — CHANGES_REQUESTED** [review]
Could you combine the commits into one?

**Comment by peternewman** [comment]
I could, or you could just use the Squash and Merge option, which is probably less effort all round:
https://docs.github.com/en/github/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits

You might need to enable it first:
https://docs.github.com/en/github/administering-a-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests

**Comment by peternewman** [comment]
Is the Squash and Merge option acceptable @ssilverman or do you want me to have a go from my end?

**Comment by ssilverman** [comment]
I'm familiar with squash merges, I just wanted to give you an opportunity to create your own commit message. If you don't want to, I'll just create a commit and give you credit in the comments.

**Comment by peternewman** [comment]
I believe if you do them from GitHub then I think it takes the Pull Request message as the message (or something equally clever/sensible) and deals with the credit too, I think we both get credited or it shows me as the author in collaboration with you or vice versa so it still properly shows my authorship, not just as a comment in the commit message.

Basically I'd rather you just did that, and it's less effort all round I believe.

I'll avoid writing the same thing on the other PRs...

**Comment by ssilverman** [comment]
That's true, but I think it also puts all the temporary commits into the message too. This is a single-file one-change commit that's in 4 commits. Could you combine them into one?

**Comment by ssilverman** [comment]
Here's the workflow that I'd like for this repo (I should really put this into a contributor's guide):
1. I'm using a rebase approach for linear history. (I'm used to working on very large codebases with lots of people, and so my habits are from that.)
2. Combine multi-commit changes that really are only one change into a single commit.
3. In the PRs, use a single commit per "thing", and put unrelated changes into different PRs.
4. Force push as necessary into your own branch to do the commit combining, so that the rebase-merge into the main branch keeps the above points.

File: .github/workflows/validate-alternative.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Validate JSONs Alternative
```
Comment (ssilverman) [inline .github/workflows/validate-alternative.yml:1]: What do you mean by "JSONs Alternative"?

Comment (peternewman) [reply .github/workflows/validate-alternative.yml:1]: Just that it was one of a number of ways of doing the validation. My first was https://github.com/peternewman/rdm-schema/tree/validation which still seems to fail, I think possibly as the validator is older than the schema you're now using.

Given the various online and CLI validators seem to catch different things, running a suite of them probably makes sense, but maybe I should name them based on which tool is doing the validation for each run.

**Comment by peternewman** [comment]
Closing this in favour of #8

**Pull request closed without merging**